src/fides/api/util/tcf/experience_meta.py

```diff
@@ -79,8 +80,14 @@ def build_experience_tcf_meta(tcf_contents: TCFExperienceContents) -> Dict:
 
     return ExperienceMeta(
         version_hash=build_tcf_version_hash(tcf_contents),
-        accept_all_fides_string=accept_all_mobile_data.IABTCF_TCString,
-        reject_all_fides_string=reject_all_mobile_data.IABTCF_TCString,
+        accept_all_fides_string=build_fides_string(
+            accept_all_mobile_data.IABTCF_TCString,
+            accept_all_mobile_data.IABTCF_AddtlConsent,
+        ),
+        reject_all_fides_string=build_fides_string(
+            reject_all_mobile_data.IABTCF_TCString,
+            reject_all_mobile_data.IABTCF_AddtlConsent,
```

Comment:
Because the AC string encodes opt-ins, for a reject all string I am returning null here.  Although there may be alternative formats desired, like "1~" but I thought null made sense.     Generally, I expect AC strings in this format "1~1.100.333", or none supplied at all.

Reply:
Updated after slack convo, going with `1~` now